### PR TITLE
Javascript/javascript1/week1

### PR DESCRIPTION
**Pull request by kinan966**
My Js homework week1

### CODE REVIEW

File: javascript/javascript1/week1/a_Dog_Age_Calculator.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+
+var dogYearOfBirth = 2017;
```
Comment (florinmaticu) [inline javascript/javascript1/week1/a_Dog_Age_Calculator.js:2]:
```suggestion
const dogYearOfBirth = 2017; 
```

File: javascript/javascript1/week1/a_Dog_Age_Calculator.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+
+var dogYearOfBirth = 2017; 
+var dogYearFuture = 2027;
+var dogYear;
+var shouldShowResultInDogYears;
+Boolean (shouldShowResultInDogYears);
```
Comment (florinmaticu) [inline javascript/javascript1/week1/a_Dog_Age_Calculator.js:6]: I don't think this line is correct. It doesn't do anything and can be removed

File: javascript/javascript1/week1/a_Dog_Age_Calculator.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+var dogYear = (dogYearFuture - dogYearOfBirth);
```
Comment (florinmaticu) [inline javascript/javascript1/week1/a_Dog_Age_Calculator.js:8]: `dogYearOfBirth` is not defined. I think you wanted to put `dogYear`

File: javascript/javascript1/week1/a_Dog_Age_Calculator.js (hunk 2 above)
Comment (kinan966) [reply javascript/javascript1/week1/a_Dog_Age_Calculator.js:6]: whiche line do u mean ? boolean?

Comment (florinmaticu) [reply javascript/javascript1/week1/a_Dog_Age_Calculator.js]: yes. the line number 6

**Review by florinmaticu — APPROVED** [review]

**Pull request merged**